src/ts/index.ts

```diff
@@ -0,0 +1,152 @@
+class parsedInput {
+  num1: string;
+  num2: string;
+  operator: string;
+
+  constructor(operator: string, num1: string, num2: string) {
```

Comment:
```suggestion
  constructor(operator: string, num1: string, num2: string):void {
```

typescript인 만큼 void일때도 명시해주는게 가독성을 위해 좋다고 생각합니다.